cmd/mt-kafka-mdm-sniff/main.go

```diff
@@ -20,6 +21,8 @@ import (
 var (
 	confFile = flag.String("config", "/etc/raintank/metrictank.ini", "configuration file path")
 	format   = flag.String("format", "{{.Part}} {{.OrgId}} {{.Id}} {{.Name}} {{.Metric}} {{.Interval}} {{.Value}} {{.Time}} {{.Unit}} {{.Mtype}} {{.Tags}}", "template to render the data with")
+
+	stdoutLock = sync.Mutex{}
```

Comment:
indeed, having it local to inputPrinter should also work fine.
But i don't see a strong reason to prefer it one way or another.